### PR TITLE
Updated each lecture with available questions, with links to the answer document; XQ

### PR DESCRIPTION
**Pull request by watercannons**

### CODE REVIEW

**Review by ykharitonova — CHANGES_REQUESTED** [review]
Please remove the link to the solutions document: they should try to solve the problem on their own and ask during the open labs or my office hours if they are unclear on how to solve it or if they got the correct answer.

File: _lectures/lec06.md [hunk 1]
```diff
@@ -168,3 +168,19 @@ int main()
...
+char grade;
+cin >> grade;
+if(grade == ‘A’)
+	cout << “Grade > 90%” << endl;
+else if(grade == ‘B’)
+	cout << “Grade > 80%” << endl;
+else if(grade == ‘C’)
+	cout << “Grade > 70%” << endl;
+else
+	cout << “Shamefur dispray” << endl;
```
Comment (ykharitonova) [inline _lectures/lec06.md]: Can this be changed to something more encouraging that helps with the "growth mindset"?

**Comment by watercannons** [comment]
Removed the links, as well as changed the switch statement example into vim commands

File: _lectures/lec06.md (hunk 1 above)
Comment (watercannons) [reply _lectures/lec06.md]: Just changed it into vim commands

**Pull request merged**